CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Next Version]
 - Implement reward-aware preference optimization.
+- Added TRT-LLM support in PPO. This can be enabled by doing `trainer.ppo.trt_llm.enable=True`. There is also aview(-1) reshard option to reshard out pipeline parallelism during inference for further speedup via `trainer.ppo.trt_llm.reshard=True`.
```

Comment:
`Added TRT-LLM support in PPO. This can be enabled by trainer.ppo.trt_llm.enable=True. There is also a reshard option to reshard out pipeline parallelism during inference(i.e running tensor and data parallel only) for further speedup via trainer.ppo.trt_llm.reshard=True.`

how is this instead?

Reply:
Better, minor formatting fixes:
```
Added TRT-LLM support in PPO. This can be enabled with `trainer.ppo.trt_llm.enable=True`. There is also a reshard option to reshard out pipeline parallelism during inference (i.e. running tensor and data parallel only) for further speedup via `trainer.ppo.trt_llm.reshard=True`.
```

Reply:
Ah, I see you actually pushed the change already, will resolve this thread and open another one to avoid confusion.